src/entity/jan_iwa/E2D730.c

```diff
@@ -1,31 +1,234 @@
 #include "common.h"
 
+extern StaticEntityData D_802BC7D0_E2F100;
+extern f32 D_802BCE20;
+// size unknown
+typedef struct structE2D730 {
+    /* 0x00 */ u8 unk_00;
+    /* 0x01 */ s8 unk_01;
+    /* 0x02 */ s8 unk_02;
+    /* 0x04 */ union { // might be 2 different structs?
+    /*      */   s16 index;
+    /*      */   f32 f;
+    /*      */ } unk_04;
```

Comment:
This is a nice way of writing unions; @ethteck thoughts about writing unions in this way in other structs?